plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/LibvirtVMDef.java

```diff
@@ -1019,21 +1021,26 @@ public String toString() {
         private String _dpdkExtraLines;
         private String _interfaceMode;
 
-        public void defBridgeNet(String brName, String targetBrName, String macAddr, NicModel model) {
-            defBridgeNet(brName, targetBrName, macAddr, model, 0);
+        /**
+         * Bridge Net
+         */
+        public void defBridgeNet(String brName, String targetBrName, String macAddr, NicModel model, Integer mtu) {
+            defBridgeNet(brName, targetBrName, macAddr, model, 0, mtu);
         }
 
-        public void defBridgeNet(String brName, String targetBrName, String macAddr, NicModel model, Integer networkRateKBps) {
+        public void defBridgeNet(String brName, String targetBrName, String macAddr, NicModel model, Integer networkRateKBps, Integer mtu) {
             _netType = GuestNetType.BRIDGE;
             _sourceName = brName;
             _networkName = targetBrName;
             _macAddr = macAddr;
             _model = model;
             _networkRateKBps = networkRateKBps;
+            _mtu = mtu;
         }
-
-        public void defDpdkNet(String dpdkSourcePath, String dpdkPort, String macAddress, NicModel model,
-                               Integer networkRateKBps, String extra, String interfaceMode) {
+       /**
+       * Dpdk Net
```

Comment:
Same here, a bit more details would be nice. Otherwise, I think that the Javadoc block could be removed.
PS: thumbs up for the Javadoc initiative it is always a plus :+1: